bootstrap/frontend/src/main/scala/io/udash/bootstrap/datepicker/UdashDatePicker.scala

```diff
@@ -274,129 +274,125 @@ object UdashDatePicker {
-                               viewMode: ViewMode = ViewMode.Days,
-                               toolbarPlacement: Option[UdashDatePicker.Placement.VerticalPlacement] = None,
-                               showTodayButton: Boolean = false,
-                               showClear: Boolean = false,
-                               showClose: Boolean = false,
-                               widgetPositioning: Option[(UdashDatePicker.Placement.HorizontalPlacement, UdashDatePicker.Placement.VerticalPlacement)] = None,
-                               widgetParent: Option[String] = None,
-                               keepOpen: Boolean = false,
-                               inline: Boolean = false,
-                               keepInvalid: Boolean = false,
-                               ignoreReadonly: Boolean = false,
-                               allowInputToggle: Boolean = false,
-                               focusOnShow: Boolean = true,
-                               enabledHours: Seq[Int] = Seq.empty,
-                               disabledHours: Seq[Int] = Seq.empty,
-                               viewDate: Boolean = false,
-                               tooltips: DatePickerTooltips = DatePickerTooltips())
+  class DatePickerOptions(
+    val format: String,
+    val dayViewHeaderFormat: String = "MMMM YYYY",
+    val extraFormats: Seq[String] = Seq.empty,
+    val stepping: Int = 1,
+    val minDate: Option[ju.Date] = None,
+    val maxDate: Option[ju.Date] = None,
+    val useCurrent: Boolean = true,
+    val collapse: Boolean = true,
+    val locale: Option[String] = None,
+    val defaultDate: Option[ju.Date] = None,
+    val disabledDates: Seq[ju.Date] = Seq.empty,
+    val enabledDates: Seq[ju.Date] = Seq.empty,
+    val icons: DatePickerIcons = new DatePickerIcons(),
+    val useStrict: Boolean = false,
+    val sideBySide: Boolean = false,
+    val daysOfWeekDisabled: Seq[DayOfWeek] = Seq.empty,
+    val calendarWeeks: Boolean = false,
+    val viewMode: ViewMode = ViewMode.Days,
+    val toolbarPlacement: Option[UdashDatePicker.Placement.VerticalPlacement] = None,
+    val showTodayButton: Boolean = false,
+    val showClear: Boolean = false,
+    val showClose: Boolean = false,
+    val widgetPositioning: Option[(UdashDatePicker.Placement.HorizontalPlacement, UdashDatePicker.Placement.VerticalPlacement)] = None,
+    val widgetParent: Option[String] = None,
+    val keepOpen: Boolean = false,
+    val inline: Boolean = false,
+    val keepInvalid: Boolean = false,
+    val ignoreReadonly: Boolean = false,
+    val allowInputToggle: Boolean = false,
+    val focusOnShow: Boolean = true,
+    val enabledHours: Seq[Int] = Seq.empty,
+    val disabledHours: Seq[Int] = Seq.empty,
+    val viewDate: Boolean = false,
+    val tooltips: DatePickerTooltips = new DatePickerTooltips()
+  ) {
+    def copy(minDate: Option[ju.Date] = minDate, maxDate: Option[ju.Date] = maxDate): DatePickerOptions = {
```

Comment:
This cannot be more error-prone and exposes our internal usage as a part of utility API. Is it really worth the unCasing?

Reply:
I'll remove this from public API.